zwave_js_server/model/value.py

```diff
@@ -181,7 +181,10 @@ def receive_event(self, event: Event) -> None:
     def update(self, data: ValueDataType) -> None:
         """Update data."""
         self.data.update(data)
-        self._value = data.get("newValue", data.get("value"))
+        if "newValue" in data:
```

Comment:
This is fine, but better would be in a future PR if we check event type so we know how to process the data.

Reply:
yes agreed.